### PR TITLE
Split qulice and codenarc profiles

### PR DESCRIPTION
**Pull request by g4s8**
#758 - use different maven profiles for `qulice` and `codenarc` updated `.rultor.yml`, `.travis/run.sh` and `README.md`

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1092](https://github.com/zerocracy/farm/issues/1092) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1092](https://github.com/zerocracy/farm/issues/1092) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1551725455); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: .rultor.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ release:
     mvn clean versions:set "-DnewVersion=${tag}" --quiet
     git commit -am "${tag}"
     cp ../settings.xml settings.xml
-    mvn clean package -Pqulice -Pzc-farm --errors --batch-mode --quiet
+    mvn clean package -Pqulice -Pcodenarc -Pzc-farm --errors --batch-mode --quiet
```
Comment (carlosmiranda) [inline .rultor.yml]: @g4s8 you should be able to shorten this a bit: `-Pqulice,codenarc,zc-farm`

File: .rultor.yml [hunk 2]
```diff
@@ -41,7 +41,7 @@ release:
 merge:
   script: |-
     pdd --source=$(pwd) --verbose --file=/dev/null
-    mvn help:system clean install -Pqulice --settings ../settings.xml
+    mvn help:system clean install -Pqulice -Pcodenarc --settings ../settings.xml
```
Comment (carlosmiranda) [inline .rultor.yml]: @g4s8 same here: `-Pqulice,codenarc`

File: .travis/run.sh [hunk 3]
```diff
@@ -1,3 +1,3 @@
 set -e
-mvn clean install -Pqulice --errors --batch-mode --quiet
+mvn clean install -Pqulice -Pcodenarc --errors --batch-mode --quiet
```
Comment (carlosmiranda) [inline .travis/run.sh]: @g4s8 same here, `-Pqulice,codenarc`

File: README.md [hunk 4]
```diff
@@ -262,8 +262,8 @@ and never hard-code any business constants.
 
 ## How to contribute
 
-Just fork it, make changes, run `mvn clean install -Pqulice`, and submit
-a pull request. Read
+Just fork it, make changes, run `mvn clean install -Pqulice -Pcodenarc`,
```
Comment (carlosmiranda) [inline README.md]: @g4s8 same here, `-Pqulice,codenarc`

**Comment by g4s8** [comment]
@carlosmiranda fixed, please look again

**Comment by g4s8** [comment]
@rultor merge

File: src/main/java/com/zerocracy/radars/github/TkGithub.java [hunk 5]
```diff
@@ -195,11 +201,11 @@ public void run() {
      */
     private static JsonObject json(final String body) {
         try (final JsonReader reader =
-            Json.createReader(
-                new ByteArrayInputStream(
-                    body.getBytes(StandardCharsets.UTF_8)
-                )
-            )) {
+                 Json.createReader(
```
Comment (carlosmiranda) [inline src/main/java/com/zerocracy/radars/github/TkGithub.java]: @g4s8 Qulice is complaining about this indentation

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1092](https://github.com/zerocracy/farm/issues/1092) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed

**Comment by ypshenychka** [comment]
@carlosmiranda Thanks!

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**